.github/workflows/golang-linux.sh

```diff
@@ -68,9 +68,21 @@ sudo kill $pid
 sleep 5
 done
 
-# Start the gin-mongo app in test omde.
+# Start the gin-mongo app in test mode.
 sudo -E env PATH="$PATH" ./../../keployv2 test -c "./ginApp" --delay 7
 
+# move keployv2 to /usr/local/bin/keploy
+mv ./../../keployv2 /usr/local/bin/keploy
+
+sed -i 's/<path for storing stubs>/\/home\/runner\/work\/keploy\/keploy\/samples-go\/gin-mongo/' main_test.go
+
+# run in mockrecord mode
```

Comment:
will this run in pipeline?

Reply:
yes

Reply:
resolve the merge conflicts

Reply:
also sceptical about this can you explain it?

Like `sed -i 's/<path for storing stubs>/home/shivamsouravjha.linux/samples-go/gin-mongo' main_test.go`
aims to replace 
The sed command you've written:
`sed -i 's/<path for storing stubs>/home/shivamsouravjha.linux/samples-go/gin-mongo' main_test.go`
aims to replace the text <path for storing stubs> with `home/shivamsouravjha.linux/samples-go/gin-mongo` in the file main_test.go.

But the current main_test.go looks like [this](https://github.com/keploy/samples-go/blob/main/gin-mongo/main_test.go) . 

So I think the main_test.go should be updated first.

either you can update it in other file and run it.

Reply:
the native-linux is branch is being checked, so the main_test.go will look like https://github.com/keploy/samples-go/blob/native-linux/gin-mongo/main_test.go